index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Contact us page</title>
+</head>
+<body>
+    <section>
+        
+    <div class="container">
+        <div class="contactinfo">
+          <div>
```

Comment:
Please use semantic HTML for cleaner and more readable code instead of wrapping everything inside divs

Reply:
ok